llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -801,6 +801,22 @@ InstructionCost VPInstruction::computeCost(ElementCount VF,
                                   cast<VectorType>(VectorTy), Mask,
                                   Ctx.CostKind, VF.getKnownMinValue() - 1);
   }
+  case VPInstruction::ActiveLaneMask: {
+    Type *Arg0Ty = Ctx.Types.inferScalarType(getOperand(0));
+    Type *Arg1Ty = Ctx.Types.inferScalarType(getOperand(1));
```

Comment:
Given `get.active.lane.mask` requires both parameters to be the same type, do we need to calculate `Arg1Ty`?

Reply:
VPTypeAnalysis should already assert that both types are the same, so would probably be good to remove here.

Reply:
Done